release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -313,6 +313,41 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
 [id="ocp-4-12-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+==== Updates to monitoring stack components and dependencies
+
+* Alertmanager to 0.24.0
+* kube-state-metrics to 2.6.0
+* node-exporter to 1.4.0
+* prom-label-proxy to 0.5.0
+* Prometheus to 2.39.1
+* prometheus-adapter to 0.10.0
+* prometheus-operator to 0.60.1
+* Thanos to 0.28.1
+
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* New
+** Added the `TelemeterClientFailures` alert, which triggers when a cluster tries and fails to submit Telemetry data at a certain rate over a period of time. The alert fires when the rate of failed requests reaches 20% of the total rate of requests within a 15-minute window.
+* Changed
+**  The `KubeAggregatedAPIDown` alert has been updated so that the time that must elapse before sending a notification is now extended from 300 seconds to 900 seconds.
+** The `NodeClockNotSynchronising` and `NodeClockSkewDetected` alerts now only evaluate metrics from the `node-exporter` job.
+** The `NodeRAIDDegraded` and `NodeRAIDDiskFailure` alerts now include a device label filter to match only the value returned by `mmcblk.p.+|nvme.+|sd.+|vd.+|xvd.+|dm-.+|dasd.+`.
+** The `PrometheusHighQueryLoad` and `ThanosQueryOverload` alerts now also trigger when a high querying load exists on the query layer.
+
+==== New option to specify pod topology spread constraints for monitoring components
+You can now use pod topology spread constraints to control how Prometheus, Thanos Ruler, and Alertmanager pods are spread across a network topology when {product-title} pods are deployed in multiple availability zones.
+
+==== New option to improve data consistency for Prometheus Adapter
```

Comment:
@jan--f this feature is also back ported to 4.10 and 4.11, do we need to mention in 4.10/4.11 doc, or only mention it in 4.12 doc is fine?
https://issues.redhat.com/browse/MON-1949

Reply:
I don't think it needs addition to the release notes, but the actual documentation should indeed be backported.

Reply:
We don't need mention backports here. However, we'll need to add it to the 4.10-11 release notes as a backported feature. 

@jan--f do you know what 4.10 and 4.11 z-stream releases included the backported feature?

Reply:
Thanks @juzhao! :point_up:

Reply:
Thanks @juzhao! I will write up a separate Jira docs issue to update the docs for those releases.